app/frontend/src/services/workspaces.ts

```diff
@@ -0,0 +1,29 @@
+import client from 'transport/ctproto-client';
```

Comment:
pls, rename `workspaces.ts` to the `workspace.ts`

Reply:
Fixed